.github/workflows/build-release.yml

```diff
@@ -3,60 +3,49 @@ name: Publish release to PyPI
 on:
   release:
     types: [published]
-
-env:
-  INSTALL_EDM_VERSION: 3.3.1
+  workflow_dispatch:
 
 jobs:
-  build-sdist:
-    runs-on: ubuntu-latest
+  build-and-publish-wheel-all:
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
 
-    steps:
-    - name: Check out the release commit
-      uses: actions/checkout@v3
-    - name: Setup EDM
-      uses: enthought/setup-edm-action@v2
-      with:
-        edm-version: ${{ env.INSTALL_EDM_VERSION }}
-    - name: Install build environment
-      run: |
-        edm --config ci/.edm.yaml install -y --version 3.8 cython numpy
-        edm run -- pip install twine
-    - name: Build sdist
-      run: edm run -- python setup.py sdist
-    - name: Publish sdist to PyPI
-      env:
-        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        edm run -- python -m twine check dist/*.tar.gz
-        edm run -- python -m twine upload dist/*.tar.gz
-
-  build-wheel-linux:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     steps:
     - name: Check out the release commit
       uses: actions/checkout@v3
-    - name: Build manylinux Python wheels
-      uses: RalfG/python-wheels-manylinux-build@v0.5.0
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
       with:
-        python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39 cp310-cp310 cp311-cp311'
-        build-requirements: 'cython numpy'
-        system-packages: 'zlib-devel'
-        pre-build-command: 'sh ci/build-wheel-deps.sh'
+        platforms: arm64
+      if: runner.os == 'Linux'
+
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.9
-    - name: Install Python packages needed for wheel upload
-      run: |
-        python -m pip install --upgrade pip setuptools
-        python -m pip install twine
-    - name: Publish wheels to PyPI
+        python-version: '3.11'
+
+    - name: Install Python packages needed for wheel build and upload
+      run: python -m pip install twine
```

Comment:
Any reason not to move this into the "Check and upload wheels" step? Now that all the build action happens inside the `cibuildwheel` step...

Reply:
That makes sense.